src/upload-tests.proj

```diff
@@ -102,10 +102,21 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyTestILCToTestHostDir" Condition="'$(BuildingUAPAOTVertical)' == 'true'">
```

Comment:
Any reason we should do this in test-runtime so that the structure always matches and is consistent with what we run in helix?